src/content/content_instance.cc

```diff
@@ -145,9 +145,8 @@ void ContentInstance::PostAsyncSuccessReply(const picojson::value& msg,
 
 void ContentInstance::HandleSyncMessage(const char* message) {
   picojson::value v;
-  picojson::value::object o;
-
```

Comment:
New commit. Even better, new pull request, as it's got nothing to do with the rest :)